GameDev/week2/homework/issues/Week 2 Homework Instructions.md

```diff
@@ -0,0 +1,15 @@
+Bitcamp Game Development - Week 2 Homework
+
+For this week’s homework, we’ll be adding assets to our game and writing basic scripts in C#.
+
+Your task is to create a road race! To do this, you’ll have to start from the project we installed in the first week (the one with the cars and the road). Try to do these steps on your own first, but if you need a little guidance, check the unity C# documentation or the recording from this week’s live session.
+
```

Comment:
same issue with styling (this is better than the live stream script formatting though)
example: https://github.com/TheDirector23/azure-functions-course/blob/main/responses/2.0-postman.md
markdown cheatsheet: https://www.markdownguide.org/cheat-sheet/